Examples/Examples-iOS/IGListKitExamples/SectionControllers/ListeningSectionController.swift

```diff
@@ -0,0 +1,58 @@
+/**
+ Copyright (c) 2016-present, Facebook, Inc. All rights reserved.
+
+ The examples provided by Facebook are for non-commercial testing and evaluation
+ purposes only. Facebook reserves all rights not expressly granted.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import IGListKit
+
+class ListeningSectionController: IGListSectionController, IGListSectionType, IncrementListener {
```

Comment:
I wouldn't mind a follow-up that does that. Tbh I really only use Swift on these example projects and am just used to hiding private stuff in implementation files. A pass of `fileprivate` or w/e on the demos would be good I think, we want to promote best-practices while not sacrificing the demo.